java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/BlockingQueueProducerProviderTest.java

```diff
@@ -54,7 +56,8 @@ void whenConstructorInvokedWithoutOffsetParameter_ThenConfigExceptionIsThrown(){
     ConfigException configException;
     try(MockedConstruction<KafkaConsumer> mockKafkaConsumer = Mockito.mockConstruction(KafkaConsumer.class)){
```

Comment:
mockKafkaConsumer variable isn't read, I believe the convention is to rename this to `ignored`

Reply:
Changed